packages/cloud-config/oem/07_cloud-metadata.yaml

```diff
@@ -0,0 +1,14 @@
+# Default cOS OEM configuration file
+#
+# This file is part of cOS and will get reset during upgrades.
+#
+# Before you change this file manually,
+# consider copying this file to /usr/local/cloud-config or
+# copy the file with a prefix starting by 90, e.g. /oem/91_custom.yaml
+name: "Cloud providers metadata"
```

Comment:
I'm not sure if we want to embed this directly :thinking:  would it behave _weirdly looking_ if booted into a baremetal?

maybe can go into a `feature, or just all commented if the purpose is to have it as a sample

Reply:
It does nothing if the provider is not found. Note this plugin is designed to be used against all supported cloud providers without having to touch it. It basically tests all listed providers one after the other until the list is over or until there is one provider that actually succeeds.